geoconvert/convert.py

```diff
@@ -163,6 +164,8 @@ def country_name_to_id(country, lang='FR'):
             countries = countries_en
         elif lang == "DE":
             countries = countries_de
+        elif lang == "PT":
```

Comment:
Maybe it is time to make a dict `{country_code: list_name}` and here do `countries = country_code_to_countries["lang"]`, instead of stacking elifs ?